src/server/scripts/Spells/spell_item.cpp

```diff
@@ -1785,6 +1791,11 @@ class spell_item_shadowmourne : public AuraScript
         }
     }
 
+    void OnRemove(AuraEffect const*, AuraEffectHandleModes)
```

Comment:
unregistered

Reply:
Fixed.

Reply:
weird param :(

Reply:
You mean comments? `AuraEffect const* /*aurEff*/, AuraEffectHandleModes /*mode*/`

Reply:
y